cartographer/mapping/pose_graph_interface.h

```diff
@@ -60,7 +60,7 @@ class PoseGraphInterface {
   PoseGraphInterface(const PoseGraphInterface&) = delete;
   PoseGraphInterface& operator=(const PoseGraphInterface&) = delete;
 
-  // Computes optimized poses.
+  // Computes optimized poses and waits for all computations to finish.
```

Comment:
I think it would be more clear to say `Waits for all computations to finish and computes optimized poses`

Reply:
+1